Dockerfile

```diff
@@ -37,99 +37,7 @@ ARG VERSION=dev
 ARG DEBUG=false
 ENV DEBUG=$DEBUG
 
-ENV DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
-ENV DOCKER_COMPOSE_VERSION=v2.5.0
-
-ENV _APP_SERVER=swoole \
-    _APP_ENV=production \
-    _APP_LOCALE=en \
-    _APP_WORKER_PER_CORE= \
-    _APP_DOMAIN=localhost \
-    _APP_DOMAIN_FUNCTIONS=functions.localhost \
-    _APP_DOMAIN_TARGET=localhost \
-    _APP_HOME=https://appwrite.io \
```

Comment:
This value isn't set anywhere else (variables.php, docker-compose.yml, compose.phtml) and will cause the `doctor` to fail on any remote host:

https://github.com/appwrite/appwrite/blob/891099c7c3dc1fe86c4e64467f9249651991b603/src/Appwrite/Platform/Tasks/Doctor.php#L248

 Let's double check all the vars with a value are set elsewhere